src/app/data-share.service.ts

```diff
@@ -1,9 +1,18 @@
+import { HttpClient } from '@angular/common/http';
 import { Injectable } from '@angular/core';
 
 @Injectable({
   providedIn: 'root'
 })
 export class DataShareService {
 
-  constructor() { }
+  constructor(private httpCall:HttpClient) { }
+
+
+callGetData(){
+  let dataResult = this.httpCall.get('https://dummyjson.com/products');
```

Comment:
add the URL in environment file or in constant variable

Reply:
fixed